### ISSUE TITLE
[VAMC] Confusing heading order on System pages

### ISSUE THREAD

**Issue by qjac** [department-of-veterans-affairs/va.gov-cms#23975]
## Describe the defect
It is a best practice to start the page heading structure with an H1. It is not a WCAG violation to start with a different heading level, but can be [confusing for AT users](https://github.com/department-of-veterans-affairs/va.gov-team/issues/132452).

Using the [Headings Map extension](chrome://extensions/?id=flbjommegcjonpdmenkdiocclhjacmbi) in Chrome, you can see that the heading in the alert and the headings in the side navigation are before the H1 in the source order. The VADS side navigation component gets around this by wrapping category headers in a div and not using heading markup. The alert heading is out of scope for this team/ticket.

## To Reproduce
1. Go to https://www.va.gov/boston-health-care/
2. Use Headings Map extension (or another tool, like VoiceOver headings rotor) to show heading hierarchy
3. Note that the page title H1 comes after headings in the side navigation.

## AC / Expected behavior
- [ ] H1 appears first in the source order. (note: If the alert heading appears first in the DOM, ignore it.) 

## Screenshots
Screenshot of VA Boston health care system page with the Headings Map report displayed:
<img width="1387" height="999" alt="Image" src="https://github.com/user-attachments/assets/5b81d9b3-90b9-4e25-b7e8-e0620bbcc84e" />

 ## Additional context
- Similar ticket: https://github.com/department-of-veterans-affairs/va.gov-team/issues/132452

## Definition of done
- [ ] Development
   - [ ] Code completed 
   - [ ] PR write up complete
   - [ ] Automated tests passed 
   - [ ] Code review requested 
   - [ ] PR approved (meets established quality and security standards)
- [ ] Review instance created/available
  - [ ] Team members notified to start QA process
- [ ] QA Process
  - [ ] A11y review (meets accessibility standards)
  - [ ] Design Review (meets design & UX requirements)
    - [ ] Tested with different devices / screen sizes
  - [ ] Functional/feature review (meets acceptance criteria)
- [ ] QA feedback reviewed/addressed, if any
- [ ] Testing completed; dev team notified to deploy
- [ ] PO/PM/external verify in prod
- [ ] Documentation updated
